relay-event-normalization/src/normalize/mod.rs

```diff
@@ -479,6 +480,71 @@ fn normalize_measurements(
     }
 }
 
+/// Computes performance score measurements
+///
+/// This computes score from vital measurements, using config options to define how it is
+/// calculated.
+fn normalize_performance_score(
+    event: &mut Event,
+    performance_score_config: Option<&PerformanceScoreConfig>,
+) {
+    if let Some(performance_score_config) = performance_score_config {
```

Comment:
nit: Might make sense to check this outside of the function, or refactor to `if let Some(performance_score_config) = performance_score_config else { return }`